### ISSUE TITLE
5ET-448 Give the popup dice roller better compatibility with the bottom ad banner (Make it sit on top of it, make it sit over it, or adjust colors, or something)

### ISSUE THREAD

**Issue by crawltracker** [5etools/tracker#552]
**Feature Request:** Give the popup dice roller better compatibility with the bottom ad banner (Make it sit on top of it, make it sit over it, or adjust colors, or something)

**Extra Information**
**Who would use it?** Everyone not using the noads site
**How would it work?** Going to put all this in the why section
**Why should this be added?** Here's a screenshot: https://puu.sh/Gq6OR/18e73a2021.png
There's multiple things going on here:
1.) The banner ad at the bottom, for my 27" screen running in 1920x1080, rests inside a semi-transparent white bar on the bottom of the screen (selected via chrome inspect. screen is squished from elements being open): https://puu.sh/Gq6R1/77d6ebb064.png
This semi-transparent bar overlaps over the dice roller unless closed. Furthermore, due to it being a semi-transparent white bar on a white background, the bar itself outside of the ad isn't immediately noticeable by a user, as the main reference window is cut so it's above the banner always. The "x" is also centered on the ad, making it less obvious that the white ad bar exists.
2.) This conflicts with the Dice Roller, which is also mostly white, and gets overlapped by that bar surrounding the ad. To the user with a wide screen it would appear that for some reason the dice roller has really faint text and it's barely readable, and it wouldn't be apparent why. They wouldn't notice darker text until they rolled more than three dice rolls, bringing the forth roll's text above the bar.  Because of the mostly-transparent design on same color background of the ad bar, it's not immediately apparent why the text in the first 3 rolls is so light while the text fourth roll is darker, and they might not know to just close the ad to remove the mostly-transparent layer.

In conclusion, this is confusing, and the dice roller and/or ad bar needs to be changed. However, you're a better web designer than me, so you probably have a better solution ¯\_(ツ)_/¯
Thanks! I enjoy the site immensely.

- BeAuMaN#1025

> Upvote - BeAuMaN#1025

> Upvote - Insaiyan7#0695

> Upvote - SigilBlue#6700

> Upvote - razvantr98#3397

> Upvote - BucketListMod#8973

> Upvote - Thundius#6394

> Downvote - Jayden1#1331

> Upvote - Modnar#2707

> Upvote - GumboVariations#7778

> Upvote - kanebeef#9509

> Downvote - zorgrath#4256

Votes: +9 / -2 / ±0

**Comment by crawltracker** [comment]
Note - Giddy#0001

Resolved - use adblock :^) Jokes aside, this isn't something I have enough control over to provide a satisfying solution for
